### DISCUSSION TITLE
custom header

### DISCUSSION THREAD

**Discussion by Pirastrino** [notiz-dev/prisma-dbml-generator#8] [Ideas]
Hi,

I'd like to use dbml-generator with [dbdocs.io](https://dbdocs.io/guide#generate-a-dbdocs-view) and a custom "header" e.g:
```javascript
Project social_blogging_site {
database_type: 'PostgreSQL'
Note: 'Social Blogging Site Database'
}
```
so after running `dbdocs build schema.dbml` I am not prompted for a project name that I am updating all the time?

**Comment by marcjulian** [comment]
@Pirastrino so you would like to specify your project and let the generator add it the `schema.dbml`

Would options like this help which would generate the `Project` block to the `schema.dbml`?

```prisma
generator dbml {
  provider   = "prisma-dbml-generator"
  projectName = "social_blogging_site"
  projectDatabaseType = "PostgreSQL"
  projectNote = "Social Blogging Site Database"
}
```

This is not yet implemented and if you like you can provide a PR to implement this feature 😄
